core/common/buffer.hpp

```diff
@@ -69,16 +69,19 @@ namespace kagome::common {
      * @brief Lexicographical comparison of two buffers
      */
     bool operator==(const Buffer &b) const noexcept;
+    bool operator!=(const Buffer &b) const noexcept;
 
     /**
      * @brief Lexicographical comparison of buffer and vector of bytes
      */
     bool operator==(const std::vector<uint8_t> &b) const noexcept;
+    bool operator!=(const std::vector<uint8_t> &b) const noexcept;
```

Comment:
maybe use boost operators to not duplicate them? 
https://www.boost.org/doc/libs/1_70_0/libs/utility/operators.htm